akita/src/internal/crud.ts

```diff
@@ -66,7 +67,7 @@ export class CRUD {
         ...state.entities,
         ...addedEntities
       },
-      ids: [...state.ids, ...addedIds]
+      ids: params && params.prepend ? [ ...addedIds, ...state.ids] : [...state.ids, ...addedIds]
```

Comment:
change:
`params && params.prepend ? [ ...addedIds, ...state.ids] : [...state.ids, ...addedIds]`
to:
`params.prepend ? [ ...addedIds, ...state.ids] : [...state.ids, ...addedIds]`

Reply:
`params.prepend` only can causing params become `undefined` and show error `cannot find property prepend of undefined`

Reply:
The default case is an empty object.

```ts
add(entities: E[] | E, params: AddParams = {} ) <====
```

Reply:
should I make it optional or default case?
`params?: AddParams` or `params: AddParams = {}`

Reply:
params: AddParams = {}